schema/url/standalone-map/plugin-floating-scatterplot.raml

```diff
@@ -0,0 +1,44 @@
+#%RAML 1.0 Library
+
+types:
+  # Type name determines the name of the Java class that will be generated.
+
+  FloatingScatterplotPostRequest:
+    type: VisualizationRequestBase
+    properties:
+      config: FloatingScatterplotSpec
+
+  FloatingScatterplotSpec:
+    additionalProperties: false
+    properties:
+      outputEntityId:
+        type: string
+      valueSpec:
+        type: string
+        enum: ['raw', 'smoothedMeanWithRaw', 'bestFitLineWithRaw']
```

Comment:
Just making a note to myself that there are potential performance issues with respect to moving around the map

1. many unaggregated points in the scatter plot (network, client rendering) - this goes against the whole ethos of server-side aggregation
2. line fitting is a kind of aggregation and avoids network+client issues but is more CPU intensive than the string-based marker geo-aggregation

So perhaps we should "ban" all non-performant visualizations for the SAM?

But that seems a bit mean.

More aggressive client-side debouncing for CPU-intensive viz types maybe?

Reply:
And maybe only allow aggregating vizs (like the fitted lines without the raw points)?

Reply:
Yes, I understand. Let's limit the number of points, or disable scatter completely, if performance is an issue.